test/functional/system/rounds.js

```diff
@@ -726,6 +726,33 @@ describe('rounds', () => {
 			});
 		});
 
+		describe('deleting last block of round twice in a row', () => {
+			before(() => {
+				return addTransactionsAndForgePromise(library, [], 0);
+			});
+
+			it('should be able to delete last block of round again', () => {
+				return deleteLastBlockPromise();
+			});
+
+			// FIXME: Unskip that test after https://github.com/LiskHQ/lisk/issues/1781 is closed
+			// eslint-disable-next-line
+			it.skip('mem_accounts table should be equal to one generated before last block of round deletion', () => {
```

Comment:
Issue #1781 is already closed.

Reply:
On `beta.8` is not skipped, but in `beta.7` we have exactly same expectation that is still skipped.